_posts/2022-01-14-code-reviews.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Reviewing finished code is too late - and other tips and motivations for code reviews
+author: lassiautio
+excerpt: >
+  Why should we do code reviews? What makes code reviews even more important than programming? Why and how to shift-left code reviews?
+  This blog post will give answers to these questions and go deeper into the world of code reviews.
+  In the end, there will be also some tips on why code reviews should be small, and why you shouldn't comment too much on code reviews.
+tags:
+ - code review
+---
+
+Code reviews are something we all have been involved, and what we do continuously in our projects. We might have contradictory feelings about them. Often they feel like just bureaucracy that is required by a manager. Sometimes we want someone to give us feedback on our code. When we see poor code, we might think that we should've reviewed the code better.
```

Comment:
we might have *mixed* feeling 

I think sounds better than *contradictory*

Reply:
Thanks! I used Google translator to find a word from Finnish. But mixed is the one that I actually was searching for.